pytorch_lightning/utilities/optimizer.py

```diff
@@ -0,0 +1,33 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
would it perhaps better fit into the already existing apply_func.py, where we have other utilities to transfer collections of data from device to device?

Reply:
the reason i added a new file with optimizers in the name was that it's not obvious `apply_func` would contain this sort of utility, which makes it harder for new users to find this.

i could move it to `apply_func` if you feel strongly about it.